src/l500/l500-depth.h

```diff
@@ -129,16 +129,24 @@ namespace librealsense
         static ivcam2::intrinsic_params get_intrinsic_params(const uint32_t width, const uint32_t height, ivcam2::intrinsic_depth intrinsic)
```

Comment:
Can we move this fn into the .cpp? Thx.

Reply:
Done, a declaration remain at the h file.